packages/components/tsconfig.json

```diff
@@ -34,6 +34,7 @@
 		{ "path": "../primitives" },
 		{ "path": "../private-apis" },
 		{ "path": "../react-i18n" },
+		{ "path": "../rich-text" },
```

Comment:
This is what I'm talking about above. I see it as a potential problem and abstraction error when a package X has dependency Y, but then Y's types depend on X's types. I'd suggest solving this by moving the corresponding rich text types to the rich text package.

Reply:
Done! Interestingly, rich-text ultimately never depended on types from components, because it was defining separate types in JSDoc instead.